src/software/thunderscope/common/proto_configuration_widget.py

```diff
@@ -36,13 +45,16 @@ def __init__(
         self.setLayout(layout)
 
         self.on_change_callback = on_change_callback
-        self.proto_to_configure = proto_to_configure
 
         # Create search query bar
         self.search_query = QLineEdit()
```

Comment:
Could you add a placeholder text for the search so it's more clear what the text box is for? E.g.
```suggestion
        # Create search query bar
        self.search_query = QLineEdit()
        self.search_query.setPlaceholderText("Search Parameters")
```

Reply:
done.